include/odp/api/spec/pool_types.h

```diff
@@ -758,6 +781,18 @@ typedef struct odp_pool_ext_param_t {
 	/** Pool type */
 	odp_pool_type_t type;
 
+	/** Parameters for user area initialization */
+	struct {
+		/** See uarea_init.init_fn of odp_pool_param_t for details
+		 *  (odp_pool_param_t::init_fn). */
+		void (*init_fn)(void *uarea, uint32_t size, void *args, uint32_t index);
+
+		/** See uarea_init.args of odp_pool_param_t for details
+		 *  (odp_pool_param_t::args). */
+		void *args;
+
+	} uarea_init;
```

Comment:
Especially these ext pool params need to document that user area initialization happen from within the final  odp_pool_ext_populate() call. Other pool param documentation (DMA and regular) could mention that callbacks are done during the pool create call.